lib/mi_bridges/driver/people_listed_page.rb

```diff
@@ -5,18 +5,70 @@ class Driver
     class PeopleListedPage < BasePage
       TITLE = "People Listed On Your Application"
 
-      delegate :primary_member, to: :snap_application
-
-      def setup; end
+      def setup
+        @member = if for_next_household_member?
+                    find_not_yet_added_member
+                  else
+                    snap_application.primary_member
+                  end
+      end
 
       def fill_in_required_fields
-        select primary_member.marital_status.titleize, from: "maritalStatus"
-        fill_in "peopleInYourHome", with: snap_application.members.size
+        select member.marital_status.titleize, from: "maritalStatus"
+
+        if for_next_household_member?
+          fill_in "firstName", with: member.first_name_and_age
```

Comment:
note that these fields are required for the primary member, but they are pre-populated after already having been added via `lib/mi_bridges/driver/personal_information_page.rb`

Reply:
Noted! It would be great to document this in a nice method name somehow, but I can't see an obvious way to do that. Any ideas?

Reply:
this is a case where TESTS would make great documentation :/